conda_forge_tick/auto_tick.xsh

```diff
@@ -447,7 +447,6 @@ def initialize_migrators(do_rebuild=False):
     add_rebuild_openssl($MIGRATORS, gx)
     add_rebuild_libprotobuf($MIGRATORS, gx)
     add_rebuild_blas($MIGRATORS, gx)
-    add_rebuild_successors($MIGRATORS, gx, 'freetype', '2.10')
```

Comment:
Hope that is OK. Waiting on this one to issue the `proj4` migration PR.
Ping @xhochy who will be doing something similar for https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/218